input/pagecontent/general-requirements.md

```diff
@@ -18,246 +18,76 @@ Implementers are advised to be familiar with the requirements of the FHIR standa
+A system that [declares conformance](declaring-conformance.html#profile-only-support) as Profile Only Support to an AU Core profile:
+- **SHALL** correctly populate a resource with all mandatory elements specified in the profile’s definition.
+- **SHALL** correctly populate a resource with all _Must Support_ elements for which a value is known. 
+- **SHALL** implement the requirements on [Missing Data](general-requirements.html#missing-data) when an element value is not known.
 
-##### Profile Only Support for a Server
-A server with AU Core Profile Only Support:
-- **SHALL** be able to populate all profile elements that are mandatory and/or labelled *Must Support* as defined by that profile's StructureDefinition.
-- **SHALL** declare support for an AU Core Profile by including its official URL in the server’s `CapabilityStatement.rest.resource.profile` or `CapabilityStatement.rest.resource.supportedProfile` element. An AU Core profile official or “canonical” URL is located on each AU Core profile page.
+A system is not required to declare Profile Only Support to all AU Core profiles. For example, a pathology laboratory system that implements the [AU Core Pathology Result Observation](StructureDefinition-au-core-diagnosticresult-path.html) profile, is not required to produce a MedicationRequest resource. 
 
+#### Profile Support + Interaction Support
 
-**Declaring Conformance**
+Profile Support + Interaction Support refers to a system that represents digital health information using the content models of AU Core profiles AND implement the REST API interactions defined by the applicable AU Core capability statement.
 
-Servers that implement Profile Only Support declare conformance to each profile supported:
-* The server hosts a [capability statement](http://hl7.org/fhir/capabilitystatement.html) at [url]/metadata that is available to both authenticated and unauthenticated clients that declares the profiles supported as either a base profile using [CapabilityStatement.rest.resource.profile](http://hl7.org/fhir/capabilitystatement-definitions.html#CapabilityStatement.rest.resource.profile) or supported profile using [CapabilityStatement.rest.resource.supportedProfile](http://hl7.org/fhir/capabilitystatement-definitions.html#CapabilityStatement.rest.resource.supportedProfile).
+An [AU Core Responder](ActorDefinition-au-core-actor-responder.html) that declares Profile Support + Interaction Support conformance:
+- **SHALL** [declare conformance](declaring-conformance.html#profile-support--interaction-support) to the [AU Core Responder Capability Statement](CapabilityStatement-au-core-responder.html) in the server's CapabilityStatement.
+- **SHALL** specify the implemented resource types, associated AU Core profiles, REST API interactions and search parameters in the server's CapabilityStatement
```

Comment:
Missing full stop at the end of a few bullet points here

Reply:
fixed.